src/collection.js

```diff
@@ -158,24 +160,26 @@ export default class Collection {
    * Adds a record to the local database.
    *
    * Options:
-   * - {Boolean} synced     Sets record status to "synced" (default: false).
-   * - {Boolean} forceUUID  Enforces record creation using any provided UUID
-   *                        (default: false).
+   * - {Boolean} synced   Sets record status to "synced" (default: false).
+   * - {Boolean} forceId  Forces the id field from the record to be used, instead
```

Comment:
I'm personally fine with `forceID` as it's properly documented, but `useRecordID` would work too. Delegating decision making to @michielbdejong.